lib/semantics/tools.cc

```diff
@@ -617,6 +618,28 @@ bool CanBeTypeBoundProc(const Symbol *symbol) {
   }
 }
 
+bool IsInitialized(const Symbol &symbol) {
```

Comment:
I don't know if IsInitialized that is not getting when DATA is being used or if is the check in check-declarations.cc.
But this test:
BLOCK DATA DBAXES
      !BLOCK DATA element ‘ifail’ at (1) must be in COMMON
      Integer ifail
      COMMON /CBAXES/I,J,K
      DATA I,J,K,ifail /1,2,3,4/
END block data DBAXES
Is passing, when it shouldn't, correct?

Reply:
We haven't implemented `DATA` statement semantics yet, but I can catch this particular case.

Reply:
Nice!